src/core/docs/api_details.md

```diff
@@ -7,11 +7,11 @@ OpenVINO Core API contains two folders:
 <pre>
  <code>
  <a href="../include/openvino">openvino/</a>                  // Common folder with OpenVINO 2.0 API
```

Comment:
```suggestion
 <a href="../include/openvino">openvino/</a>                  // Common folder with OpenVINO API
```

Reply:
So all mentions of "OpenVINO API 2.0" (and their variations) should be replaced by "OpenVINO API"?

Reply:
yes, because we now have only 1 API :)

Reply:
done